core_backend/app/users/models.py

```diff
@@ -25,6 +24,7 @@ class UserDB(Base):
 
     user_id: Mapped[str] = mapped_column(String, primary_key=True, nullable=False)
     username: Mapped[str] = mapped_column(String, nullable=False)
+    hashed_password: Mapped[str] = mapped_column(String(64), nullable=False)
```

Comment:
Added hashed_passwords column temporarily - once we have OAuth we won't be storing passwords.